web/src/test/java/no/nav/modiapersonoversikt/service/saker/SakerServiceImplTest.kt

```diff
@@ -115,6 +167,33 @@ class SakerServiceImplTest {
         assertThat(saker[0].finnesIGsak, `is`(false))
     }
 
+    @Test
+    fun `legger til oppfolgingssak fra Arena dersom denne ikke finnes i sak med unleash true`() {
+        every { safService.hentSaker(any()) } returns GraphQLResponse()
+        every { unleashService.isEnabled(Feature.BRUK_ARENA_SAK_VEDTAK_SERVICE) } returns true
+        val saksId = "123456"
+        val dato = LocalDate.now().minusDays(1)
+        val xmlDato = DatatypeFactory.newInstance().newXMLGregorianCalendar(dato.toString())
+
+        every { arenaSakVedtakService.hentSaksInfoListeV2(any(), any(), any(), any(), any(), any(), any()) } answers {
+            val sakslisteholder = arg<Holder<SaksInfoListe>>(6)
+            sakslisteholder.value.withSaksInfo(
+                SaksInfo()
+                    .withSaksId(saksId)
+                    .withSakstypekode("ARBEID")
+                    .withSakOpprettet(xmlDato)
+                    .withTema(TEMAKODE_OPPFOLGING)
+            )
+        }
```

Comment:
Så hvordan fungerte dette egentlig? For hvert kall til `hentSaksInfoListeV2` så kjøres en kodeblokk som gir en oppdatert saksliste-Holder?

Reply:
Slik jeg forstod det: Når man bruker answers, noe vi må gjøre i dette tilfellet her siden funksjonen vår er en void, så spesifiserer man hva som skal skje når den mockede funksjonen kalles. Så på den måten så kan vi spesifisere at når denne funksjonen kalles så skal saksliste oppdateres med disse verdiene her. Det blir som når man har en funksjon som returnerer en verdi, så bruker vi returns for å spesifisere at dette er verdien den mockede funksjonen skal returnere når den blir kalt.